lib/cashbox-batches.js

```diff
@@ -0,0 +1,18 @@
+const db = require('./db')
+
+function createCashboxBatch (rec) {
+  const sql = 'INSERT INTO cashbox_batches (device_id, created) VALUES ($1, now()) RETURNING *'
+  const sql2 = ` 
+    UPDATE bills SET cashbox_batch_id=$1
+    FROM cash_in_txs
+    WHERE bills.cash_in_txs_id = cash_in_txs.id AND
+          cash_in_txs.device_id = $2 AND 
+          bills.cashbox_batch_id IS NULL
+  `
+  return db.oneOrNone(sql, [rec.deviceId])
+    .then(res => {
+      return db.oneOrNone(sql2, [res.id, res.device_id])
+    })
+}
```

Comment:
Queries executed in sequence that depend on the returned values of the previous, should be done either inside db.tx or db.task
My opinion is that this should be done inside a transaction (db.tx), because in case of error on the second query, a rollback happens and no change caused by the first query occurs
https://vitaly-t.github.io/pg-promise/Database.html#tx